modules/representation_management/spec/requests/representation_management/v0/pdf_generator_2122a_swagger_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+require Rails.root.join('spec', 'rswag_override.rb').to_s
+require_relative '../../../support/swagger_shared_components/v0'
+
+RSpec.describe 'PDF Generator 21-22a', openapi_spec: 'modules/representation_management/app/swagger/v0/swagger.json',
+                                       type: :request do
+  before do
+    create(:accredited_individual, :with_phone,
+           id: SwaggerSharedComponents::V0.representative[:id])
+  end
+
+  path '/representation_management/v0/pdf_generator2122a' do
+    post('Generate a PDF for form 21-22a') do
+      tags 'PDF Generation'
+      consumes 'application/json'
+      produces 'application/pdf'
+      operationId 'createPdfForm2122a'
+
+      parameter SwaggerSharedComponents::V0.body_examples[:pdf_generator2122a_parameter]
+
+      response '200', 'PDF generated successfully' do
+        let(:pdf_generator2122a) do
+          SwaggerSharedComponents::V0.body_examples[:pdf_generator2122a]
+        end
+        run_test!
+      end
+
+      response '422', 'unprocessable entity response' do
+        let(:pdf_generator2122a) do
+          params = SwaggerSharedComponents::V0.body_examples[:pdf_generator2122a]
+          params[:veteran][:name].delete(:first)
+          params
+        end
+        schema '$ref' => '#/components/schemas/Errors'
```

Comment:
is there a way to show a specific error message rather than "string"? I think providing a real example would be good.

The 2122 swagger docs had an example at one time, but now it shows "string" too.

Reply:
Yep, I put the real example of 
```
"errors": [
           "Veteran first name can't be blank"
         ]
```
in there.  I updated the screenshot above.